doc/src/readme.m

```diff
@@ -6,6 +6,64 @@
 ..esc ~
 
 
+<Version 6.69> (12/06/2023)>
```

Comment:
Text should be in English. 
Don't forget that Yannick will give a hand starting from 2024.

Reply:
I know. I chose French because (almost) currently all the doc is in French. Should I switch to English for the new versions ?

Reply:
You may ask chatGPT for translating the text ;)

Reply:
> I know. I chose French because (almost) currently all the doc is in French. Should I switch to English for the new versions ?

Yes please.